src/api_calls/orders_calls.js

```diff
@@ -51,3 +51,28 @@ export async function getAllOrders()
         });
 }
 
+export function DeleteOrder(id){
```

Comment:
You can rename this function to `deleteOrder`.

Reply:
ok